.changeset/unlucky-birds-pay.md

```diff
@@ -0,0 +1,6 @@
+---
+"vue-demo-store": minor
+---
+
+- Added `refreshShoppingAndPaymentMethods` method that is used for rules scenarios that are related to the shipping or payment
```

Comment:
shouldn't be `shipping` not `shopping`?
```suggestion
- Added `refreshShippingAndPaymentMethods` method that is used for rules scenarios that are related to the shipping or payment
```
also in the code

Reply:
ups, done, thank you